### ISSUE TITLE
Set expiresIn

### ISSUE THREAD

**Issue by Tienisto** [jonasroussel/jsonwebtoken#8]
I cannot set the expiresIn (`exp`) attribute. Do I use the library correctly?

Here is my code piece:

```dart
String jwt = JWT({}).sign(
    SecretKey('somejwtsecretkeywhichijustmadeup'),
    algorithm: JWTAlgorithm.HS256,
    expiresIn: const Duration(days: 10)
);
```

**Comment by jonasroussel** [comment]
Yes, this is exactly how it should be used. Its a compilation error ? The generated jwt body's does not contains `exp` attribute ?
I ran my example code with an expiresIn, and its work perfectly.

**Comment by Tienisto** [comment]
I think, that's because I used the `{}` argument. Using `{'dummy': 'dummy'}` works

**Comment by jonasroussel** [comment]
Ok ! I know why ! I will fix this soon.

**Comment by jonasroussel** [comment]
Fixed ! You can now use the `v2.0.1`
